### DISCUSSION TITLE
What do you think about daScript?

### DISCUSSION THREAD

**Discussion by me2beats** [godotengine/godot-proposals#4653] [Scripting]
https://dascript.org/
Looks like python/gdscript
They say it's very fast.
Can it be an alternative to gdscript?

**Comment by Calinou** [comment]
[One of the reasons GDScript was created is to have complete control over language decisions](https://docs.godotengine.org/en/stable/about/faq.html#doc-faq-what-is-gdscript). *Replacing* GDScript with a third-party language would be counterproductive to that goal. C# is the only other officially supported language because it's ubiquitous by now, along with C++ (with GDExtension).

That said, anyone is free to create a GDExtension language binding for daScript (or any other language).

> They say it's very fast.

First-party benchmarks should always be taken with a grain of salt, at least until you verify them on your own hardware (if reproduction instructions are provided) :slightly_smiling_face:
